include/aws/io/private/event_loop_impl.h

```diff
@@ -0,0 +1,342 @@
+#ifndef AWS_IO_EVENT_LOOP_IMPL_H
+#define AWS_IO_EVENT_LOOP_IMPL_H
+
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/io/io.h>
+
+#include <aws/common/atomics.h>
+#include <aws/common/hash_table.h>
+#include <aws/common/ref_count.h>
+
+AWS_PUSH_SANE_WARNING_LEVEL
+
+struct aws_event_loop;
+struct aws_overlapped;
+
+typedef void(aws_event_loop_on_completion_fn)(
```

Comment:
It seems that event loop related members are all moved to private header, are we no longer exposing the event loop API? Would that be a breaking change as well? I did a quick search, we used `aws_event_loop` in multiple c libraries.

Reply:
What now-private APIs are we using?

Reply:
`aws_event_loop_new_default` is used in c-http tests to create an event loop group. I guess we can remove it from c-http and use the new event loop group API anyway.